### DISCUSSION TITLE
Ubuntu Bionic (18.04) installation problems

### DISCUSSION THREAD

**Discussion by iilme** [atauenis/webone#140] [Q&A]
Followed the webone installation wiki for the 18.04 Ubuntu version.

Got stuck with dotnet-runtime-6.0 which requires (among a plethora of libraries that won't upgrade past Bionic's latest versions) libicu74, which is a 22.04+ package

I'm fine with using a previous version of WebOne, I just don't know which one to use. 0.17.1 doesn't install, nor does 0.17.0.

I can't update Ubuntu because of hardware constraints.

**Comment by atauenis** [comment]
Dotnet for Ubuntu pre-22.04 is hosted in Microsoft repository. So after end of Bionic support, there are no guarantees about compatibility of newer updates with Ubuntu 18.04. Try older versions of dotnet-runtime-6.0 and its dependencies from here: https://packages.microsoft.com/ubuntu/18.04/prod/pool/main/d/ 

Also it will be need to hold versions via APT to prevent updating of them from Microsoft repository. Or just remove `packages.microsoft.com` line from `/etc/apt/sources.list` (`sources.list.d/`).

> I'm fine with using a previous version of WebOne, I just don't know which one to use. 0.17.1 doesn't install, nor does 0.17.0.

All versions of WebOne since 0.12.0 are using .NET 6. And 0.11.3 is a very, very old version right now (but it used .NET 3.1 to run).
